src/app/wishlists/wishlist-file.ts

```diff
@@ -85,6 +91,51 @@ export function toWishList(...fileTexts: string[]): WishListAndInfo {
   }
 }
 
+function autoUpgradeRoll(roll: WishListRoll): WishListRoll[] {
```

Comment:
Mind documenting this?

Reply:
I added documentation (and linked #8218 for maybe more context?).